packages/search/test/autodetect.test.ts

```diff
@@ -299,3 +362,185 @@ export { Foo as One, Bar }
     errors: [],
   });
 });
+
+test("gets component imports for longer arrow statements", () => {
+  const contents = `
+export const One = () => {
+  const a = 1;
+  return <div />;
+};
+`.trim();
+
+  const componentImport = autodetectComponentExports(
+    contents,
+    "/path/to/file.ts",
+  );
+
+  expect(componentImport).toEqual({
+    filepath: "/path/to/file.ts",
+    fileExports: [
+      {
+        name: "One",
+        isDefaultExport: false,
+        source: `{
+  const a = 1;
+  return <div />;
+}`,
+      },
+    ],
+    errors: [],
+  });
+});
+
+test("gets component imports for longer functions", () => {
+  const contents = `
+export function One () {
+  const a = 1;
+  return <div />;
+};
```

Comment:
Maybe run this through `prettier`. I think it'll remove the space before the `(`.